torch/jit/operator_upgraders.py

```diff
@@ -53,7 +53,7 @@ def div__Scalar_0_3(self: torch.Tensor, other: Union[int, float]) -> torch.Tenso
     # type here.
     @no_type_check
     @torch.jit.script
-    def full_names_0_4(size: List[int], fill_value: Union[int, float], *,
+    def full_0_4(size: List[int], fill_value: Union[int, float], *,
```

Comment:
this is done to make sure our code works end-to-end. This shouldn't be here once we merge https://github.com/pytorch/pytorch/pull/69860

Reply:
resolved